cdr/resolver.go

```diff
@@ -185,24 +184,16 @@ func marshalCanonicalCAASet(set []*dns.CAA) ([]byte, error) {
 
 // LookupCAA performs a multipath CAA DNS lookup using GPDNS
 func (cdr *CAADistributedResolver) LookupCAA(ctx context.Context, domain string) ([]*dns.CAA, error) {
-	query := make(url.Values)
-	query.Add("name", domain)
-	query.Add("type", strconv.Itoa(int(dns.TypeCAA))) // CAA
-	rawQuery := query.Encode()
```

Comment:
Mainly it was just a extra allocation that was unnecessary when building a relatively simple query string.